.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,6 @@
+# Issue: Support
+
+## Ask your question clearly and concisely (Opportunity for detail is below)
```

Comment:
hmm.. not sure about the wording here.. seems like we don't trust our customers to provide a concise issue title..

Reply:
That's fair, will leave it at 'Ask your question', I taken inspiration from a couple existing templates but I think you're right.